### DISCUSSION TITLE
Permasalahan install packages di R

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#387] [Q&A] [answered]
Dear tim algoritma,

Mohon bantuannya saya mendapatkan permasalahan pada saat install beberapa packages, antara lain:

- rattle
- caret
- dplyr

Untuk permasalahan  dari packages `dplyr` saya sudah melakukan langkah2 sbb:
1. Restart R
2. remove.packages ("rlang")
3. remove.packages ("dplyr")
4. install.packages ("rlang")
5. install.packages ("dplyr")

Hasil yang saya dapatkan masih error, mohon saran dan petunjuknya.

Best regards,
JT
<img width="776" alt="image" src="https://user-images.githubusercontent.com/103427989/193786183-dbb7c8be-163f-4b32-864c-f914f59279ab.png">

<img width="674" alt="image" src="https://user-images.githubusercontent.com/103427989/193782169-9f69ae7b-601a-42aa-905e-dfa6a09a8034.png">

<img width="763" alt="image" src="https://user-images.githubusercontent.com/103427989/193780819-c4e4e348-ef32-4ecf-adf0-3b8e5107f679.png">

**Comment by fafilia** [comment, marked as answer]
Dear @joejnt ,
Jika diperhatikan untuk errornya menunjukkan bahwa import library dplyr membutuhkan versi library rlang >=1.0.2 sementara yang terinstall di device Bapak adalah versi rlang 1.0.1 yang mana ini belum sesuai dengan syarat yang dibutuhkan. Oleh karena itu, bisa di uninstall kembali package rlang kemudian install ulang dengan versi yang terbaru dengan mengikuti langkah-langkah pada link berikut:
https://stackoverflow.com/questions/53542831/how-to-install-specific-version-of-rlang-package-in-r

Silahkan untuk cek juga source dari library rlang yang akan diinstall pada link berikut: https://cran.r-project.org/src/contrib/Archive/rlang/

Jika sudah boleh direstart R-nya dan boleh dicek kembali apakah library dplyr sekarang sudah bisa diimport. Sebagai tambahan, apabila versi rlang >=1.0.2 tidak bisa diinstall pada versi R yang ada di device Anda, silahkan untuk update R.
